.github/stale.yml

```diff
@@ -4,19 +4,28 @@ daysUntilStale: 180
 daysUntilClose: 7
 # Issues with these labels will never be considered stale
 exemptLabels:
-  - Never Stale
+  - Roadmap
 # Label to use when marking an issue as stale
 staleLabel: Stale
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
   This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed in 7 days if no further activity occurs. 
-  If you feel this is incorrect please comment to keep it alive, with a reason
-  why. 
+  activity in 6 months. It will be closed in 7 days if no further activity occurs. 
+
+  Allowing issues to close as stale helps us filter out issues which can wait
+  for future development time. All issues can be reopened at any point if your
```

Comment:
Is it worth adding something to make it super clear that issues still exist and can be searched and commented on etc, i.e. they haven't "gone away", it's just a different way of filtering them?

Reply:
Good shout, I've re-jigged the wording slightly